crates/cast/src/main.rs

```diff
@@ -240,3 +244,51 @@ fn update_cast_config(config: &mut CastConfig, cli: &Cli) {
 
     config.accounts_file = Utf8PathBuf::from(shellexpand::tilde(&new_accounts_file).to_string());
 }
+
+async fn get_account<'a>(
```

Comment:
I'd rather move this function away to lib.

All the keystore functionality could be placed in `helpers/keystore.rs` and imported where needed, this function should (ideally) just pick keystore or accounts file based on user input.

Reply:
Problem with this is it takes in `Cli` and `CastConfig`, which makes it work only in main. I could tweak the inputs however

Reply:
We could just have separate `get_account` function per type (file/keystore), import it in main and do a simple if or something.